### DISCUSSION TITLE
Roadmap for version 0.8.0

### DISCUSSION THREAD

**Discussion by bearyung** [AvaloniaUtils/DialogHost.Avalonia#60] [General]
Hello,

I would like to know if there's any roadmap / timeline for making the 0.8.0-avalonia11dot1-3 to a production release? 

As my company's policy does not allow to use pre-release library in the production release, but one of the fix ([Add the .disable-content-cover-animations class])(https://github.com/AvaloniaUtils/DialogHost.Avalonia/commit/3ce657c961faba0351cc4019973cec866b8c2ffd) is being fixed in this pre-release version.

Thank you for your help.

bearyung

**Comment by SKProCH** [comment]
Hello, i'll try to find time to publish tomorrow.

**Reply by bearyung** [reply]
Thanks for your help and prompt response! 
cheers!

**Comment by SKProCH** [comment]
Version 0.8.0 published on nuget,
